wenet/transformer/subsampling.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
         self.right_context = 0
         self.subsampling_rate = 1
 
-    def position_encoding(self, offset: int, size: int) -> torch.Tensor:
+    def position_encoding(self, offset: torch.Tensor, size: int) -> torch.Tensor:
```

Comment:
We can directly use scalar type in onnx exporting  with opset_version>=11, please see https://pytorch.org/docs/stable/onnx.html

Reply:
@robin1001  Yes, scalar is supportted. But for batching requests, we need to give each request an offset. It is ok for offline onnx model but not suitable for streaming.